packages/@webex/test-helper-chai/src/index.js

```diff
@@ -8,6 +8,9 @@ const sinon = require('sinon');
 const registerAssertions = require('./assertions');
 
 chai.use(registerAssertions);
+
+chai.config.truncateThreshold = 0;
```

Comment:
ah, thanks for this! I didn't know it was configurable, it's been so annoying when it got truncated

Reply:
yeah, it took a while to figure out but i was so glad i was able to fix it.
the config truncates it by default...